src/osemosys/schemas/model.py

```diff
@@ -38,14 +36,17 @@ class RunSpec(OSeMOSYSBase):
     #transmission_technologies: List[TechnologyTransmission]
 
     # reserve margins if any
-    reserve_margins_commodity: List[Dict[str,RegionYearData]] | None
-    reserve_margins_technology: List[Dict[str, RegionYearData]] | None
+    reserve_margins_level: RegionYearData | None
+    reserve_margins_commodity: RegionCommodityYearData | None
+    reserve_margins_technology: RegionTechnologyYearData | None
 
     # renewable targets
     renewable_targets: RegionYearData | None
     
+    # Default values
```

Comment:
hmmm let's think about where we want defaults to come from. Maybe we want some bounds in the schemas, but this schema is supposed to express a fully-formed model-run, so probably shouldn't inject too much other data